### ISSUE TITLE
neovim nightly failed to start on Win10

### ISSUE THREAD

**Issue by it3q** [neovim/neovim#14577]
neovim nightly (version NVIM v0.5.0-dev+1326-ge0a01bdf7) is unable to start on Windows 10. Both nvim-qt.exe and nvim.exe reports the same error (See attached file). I tried the nightly few days ago and got the same error.

OS version: Windows 10.0.17763

I simply download, unpack and tried to execute the exe files under bin directory. The last neovim version that I have and runs on my machine is:

```
NVIM v0.5.0-dev+1293-ga0da4c3a4
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3
Compilation: C:/Program Files (x86)/Microsoft Visual Studio/2017/Enterprise/VC/Tools/MSVC/14.16.27023/bin/Hostx86/x64/cl.exe /DWIN32 /D_WINDOWS /W3 /MD /Zi /O2 /Ob1 /DNDEBUG /W3 -D_CRT_S
ECURE_NO_WARNINGS -D_CRT_NONSTDC_NO_DEPRECATE -DWIN32 -D_WIN32_WINNT=0x0600 -DINCLUDE_GENERATED_DECLARATIONS -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -DMIN_LOG_LEVEL=3 -ID:/a
/neovim/neovim/build/config -ID:/a/neovim/neovim/src -ID:/a/neovim/neovim/nvim-deps/usr/include -ID:/a/neovim/neovim/build/src/nvim/auto -ID:/a/neovim/neovim/build/include
Compiled by runneradmin@fv-az152-644

Features: -acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM\sysinit.vim"
  fall-back for $VIM: "C:/Program Files/nvim/share/nvim"
```

![neovim_unable_to_start](https://user-images.githubusercontent.com/83685422/118587382-cf8f7a80-b79c-11eb-8b98-c214108f0693.png)

**Comment by jamessan** [comment]
Without more information, there's not much we can do.  I'm able to run the nightly just fine on Windows 10 Version 10.0.18363.1500.  From what I've seen from a brief search about that error-code, it sounds more like an issue with your computer than with the nvim binary.

**Comment by janlazo** [comment]
Searching for `a0da4c3a4` leads to https://github.com/neovim/neovim/pull/14334 which is prior to my runtime PR, https://github.com/neovim/neovim/pull/14424. Maybe a vim patch PR like https://github.com/neovim/neovim/pull/14403 introduced a regression in your envirionment.

**Comment by it3q** [comment]
Thank you for your time and effort! I found the root cause of this issue. It is not a neovim bug. Apologies for the bug report!

Recently I started to use cygwin tools more frequently, so I unpacked nvim-win64.zip with cygwin's unzip tool. Although in terms of file content it decompressed the archive just like 7zip, the access rights was different. *.dll files had access code 666 while in the version which was unpacked with 7zip 770. After fixing this, neovim started.

`unzip nvim-win64.zip
cd Neovim
find -name '*.dll' -exec chmod 770 {} \;`
